common/commands_test.go

```diff
@@ -1,18 +1,19 @@
 package common
 
 import (
-	"os"
 	"testing"
 
-	"gotest.tools/assert"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestCommandMock(t *testing.T) {
 	defer CleanupEnvironment()()
```

Comment:
just in case, `CleanupEnvironment` is not needed when `os.Setenv` is used

Reply:
done